internal/internal_workflow_client.go

```diff
@@ -1932,7 +1944,7 @@ func (ch *completedUpdateHandle) Get(ctx context.Context, valuePtr interface{})
 
 func (luh *lazyUpdateHandle) Get(ctx context.Context, valuePtr interface{}) error {
 	enc, err := luh.client.PollWorkflowUpdate(ctx, luh.ref)
-	if err != nil {
+	if err != nil || valuePtr == nil {
```

Comment:
Why is this needed?

Reply:
Without it, [this](https://github.com/cludden/sdk-go/commit/8e8c57779096726de0f671d19365b6cbfa648e04#diff-8118e44ac15fb5ab2e2ece2ab52fd23062a10b5cc9c55ee5b3620a1242cff27cR1370) fails with `payload item 0: type: <nil>: not a pointer type`

Reply:
Hm OK ideally `EncodedValue.Get` handles this case, but that's out of scope for this PR

Reply:
I pulled this part out to include in the next release for the Go SDK https://github.com/temporalio/sdk-go/commit/86764f86bf3066a40097d208ed32b336afa3ab7f